builder/builder/gce.go

```diff
@@ -134,23 +135,10 @@ func (s *Server) NewGCEService(ctx context.Context) error {
 	return nil
 }
 
-// RandomName generates a random name for the GCE VM.
-func RandomName() string {
-	const letterBytes = "abcdefghijklmnopqrstuvwxyz"
-	const n = 6 // letters long
-
-	mrand.Seed(time.Now().UnixNano())
-	b := make([]byte, n)
-	for i := range b {
-		b[i] = letterBytes[mrand.Intn(len(letterBytes))]
-	}
-	return fmt.Sprintf("%s-%s", instanceNamePrefix, b)
-}
-
-// NewInstance starts a Windows VM on GCE and returns host, username, password.
-func (s *Server) NewInstance() error {
+// newInstance starts a Windows VM on GCE and returns host, username, password.
+func (s *Server) newInstance() error {
 	scmd := startupCmd // TODO: find better way to take address of const
-	name := RandomName()
+	name := uuid.New()
```

Comment:
Understand the need to replace hand-rolled code with standard UUID implementation, but if someone (not necessarily the same user) happens to glance at the GCE dashboard there's no clue about what the host is doing.  How about fmt.Sprintf("windows-builder-%s", uuid.New()) or similar?